lightly/embedding/_base.py

```diff
@@ -9,7 +9,7 @@
 import omegaconf
 from omegaconf import DictConfig
 from pytorch_lightning import LightningModule, Trainer
-from pytorch_lightning.callbacks.callback import Callback
+from pytorch_lightning.callbacks import Callback
```

Comment:
Importing directly from `pytorch_lightning.callbacks` is backwards compatible with older lightning versions